tests/Tezos/TransactionTests.cpp

```diff
@@ -21,7 +20,7 @@ TEST(TezosTransaction, forgePayment) {
     257,
     1,
     Address("tz1Yju7jmmsaUiG9qQLoYv35v5pHgnWoLWbt"),
-    type
+    TransactionType::PAYMENT
```

Comment:
Tezos calls these `transactions` and it gets serialized to a field called `kind`. 

I'd expected something like `Kind::TRANSACTION`.

Also, why not just use the enum on the proto?

Reply:
good point!